lib/ex_json_schema/validator/error/string_formatter.ex

```diff
@@ -22,9 +22,7 @@ defmodule ExJsonSchema.Validator.Error.StringFormatter do
 
   defimpl String.Chars, for: Error.AllOf do
     def to_string(%Error.AllOf{invalid: invalid}) do
-      "Expected all of the schemata to match, but the schemata at the following indexes did not: #{
-        Enum.map_join(invalid, ", ", & &1.index)
-      }."
+      "Expected all of the schemata to match, but the schemata at the following indexes did not: #{Enum.map_join(invalid, ", ", & &1.index)}."
```

Comment:
Could it be you have different formatter settings? The line should break at 120 characters so the previous formatting looks correct to me 🤔

Reply:
I am using the defaults of mix format!

Reply:
I have added a config entry to expand the line length.